4-projects/business_unit_2/development/example_base_shared_vpc_project.tf

```diff
@@ -30,6 +30,10 @@ module "base_shared_vpc_project" {
   sa_roles                    = ["roles/editor"]
   enable_cloudbuild_deploy    = true
   cloudbuild_sa               = var.app_infra_pipeline_cloudbuild_sa
+  activate_apis = [
+    "iam.googleapis.com",
+    "cloudresourcemanager.googleapis.com"
```

Comment:
I am assuming you ran into an error without these enabled?

Reply:
Yes, we got errors in the `plan` and `apply` for the step `5-app-infra` code

```
Step #1 - "tf plan validate all": *************** TERRAFORM PLAN *******************
Step #1 - "tf plan validate all":       At environment: business_unit_1/non-production 
Step #1 - "tf plan validate all": **************************************************
Step #1 - "tf plan validate all": Refreshing Terraform state in-memory prior to plan...
Step #1 - "tf plan validate all": The refreshed state will be used to calculate this plan, but will not be
Step #1 - "tf plan validate all": persisted to local or remote state storage.
Step #1 - "tf plan validate all": 
Step #1 - "tf plan validate all": module.base_shared_gce_instance.module.instance_template.data.google_compute_image.image: Refreshing state...
Step #1 - "tf plan validate all": data.google_active_folder.env: Refreshing state...
Step #1 - "tf plan validate all": module.base_shared_gce_instance.module.instance_template.data.google_compute_image.image_family: Refreshing state...
Step #1 - "tf plan validate all": 
Step #1 - "tf plan validate all": Error: Error when reading or editing Folder Not Found : fldr-non-production: googleapi: Error 403: Cloud Resource Manager API has not been used in project 829608620091 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=829608620091 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry., accessNotConfigured
Step #1 - "tf plan validate all": 
Step #1 - "tf plan validate all":   on main.tf line 19, in data "google_active_folder" "env":
Step #1 - "tf plan validate all":   19: data "google_active_folder" "env" {
Step #1 - "tf plan validate all": 
Step #1 - "tf plan validate all": 
Finished Step #1 - "tf plan validate all"
```

and 

```
Step #4 - "tf apply": *************** TERRAFORM APPLY *******************
Step #4 - "tf apply":       At environment: business_unit_1/non-production 
Step #4 - "tf apply": ***************************************************
Step #4 - "tf apply": module.base_shared_gce_instance.google_service_account.compute_engine_service_account: Creating...
Step #4 - "tf apply": 
Step #4 - "tf apply": Error: Error creating service account: googleapi: Error 403: Identity and Access Management (IAM) API has not been used in project 829608620091 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/iam.googleapis.com/overview?project=829608620091 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry., accessNotConfigured
Step #4 - "tf apply": 
Step #4 - "tf apply":   on ../../modules/env_base/main.tf line 21, in resource "google_service_account" "compute_engine_service_account":
Step #4 - "tf apply":   21: resource "google_service_account" "compute_engine_service_account" {
Step #4 - "tf apply": 
Step #4 - "tf apply": 
Finished Step #4 - "tf apply"
```

Reply:
Got it, thanks for clarifying